src/main/java/com/microsoft/sqlserver/jdbc/Parameter.java

```diff
@@ -531,19 +531,14 @@ private void setTypeDefinition(DTV dtv) {
                             param.typeDefinition = SSType.DECIMAL.toString() + "(" + valueLength + "," + scale + ")";
                         }
                     } else {
-                        BigDecimal bigDecimal = null;
+                        BigDecimal bigDecimal;
                         if (dtv.getJavaType() == JavaType.BIGDECIMAL
                                 && null != (bigDecimal = (BigDecimal) dtv.getSetterValue())) {
```

Comment:
since I have comments in other places, can we move this assignment out of the 'if'

Reply:
Moved.